.storybook/stories/tabs/tabs-actions.stories.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrTabs, ClrTabsModule } from '@clr/angular';
+import { moduleMetadata, StoryFn, StoryObj } from '@storybook/angular';
+
+import { TabsLayout } from '../../../projects/angular/src/layout/tabs/enums/tabs-layout.enum';
+
+export default {
+  title: 'Tabs/Tabs Actions',
+  decorators: [
+    moduleMetadata({
+      imports: [ClrTabsModule],
+    }),
+  ],
+  component: ClrTabs,
+  argTypes: {
+    // inputs
+    clrLayout: { control: 'inline-radio', options: TabsLayout },
+    // methods
+    closeOnEscapeKey: { control: { disable: true }, table: { disable: true } },
+    closeOnFocusOut: { control: { disable: true }, table: { disable: true } },
+    closeOnOutsideClick: { control: { disable: true }, table: { disable: true } },
+    openOverflowOnFocus: { control: { disable: true }, table: { disable: true } },
+    resetKeyFocusCurrentToActive: { control: { disable: true }, table: { disable: true } },
+    toggleOverflowOnClick: { control: { disable: true }, table: { disable: true } },
+    toggleOverflowOnPosition: { control: { disable: true }, table: { disable: true } },
+    // story helpers
+    createArray: { control: { disable: true }, table: { disable: true } },
+    clickTabAction: { control: { disable: true }, table: { disable: true } },
+    tabCount: { control: { type: 'number', min: 1, max: 100 } },
+    activeTab: { control: { type: 'number', min: 1, max: 100 } },
+  },
+  args: {
+    // inputs
+    clrLayout: TabsLayout.HORIZONTAL,
+    // story helpers
+    createArray: n => new Array(n),
+    clickTabAction: () => alert('Tab action clicked!'),
+    tabCount: 4,
+    activeTab: 1,
+    title: 'Tab',
+    content: 'Tab Content',
+  },
+};
+
+const tabsTemplate: StoryFn = args => ({
+  template: `
+    <clr-tabs [clrLayout]="clrLayout">
+      <clr-tabs-actions position="right">
```

Comment:
Viasual tests for `left` position is missiing.

Reply:
Resolved in [2d0a306](https://github.com/vmware-clarity/ng-clarity/pull/1620/commits/2d0a306495298892d43ee3dd569062800ee44180)